.travis/filter/repr.py

```diff
@@ -0,0 +1,25 @@
+from __future__ import absolute_import, division, print_function
```

Comment:
Is this ansible plugin executed by python 2?
Otherwise these future imports are not needed.

Reply:
This is common ansible boilerplate. And you never know if someone is trying to some of this with python2.
(You should be able to use the playbooks provided here in a local vm to debug those containers for example.)